int32 uav_id
float32 src_id
float32 dst_id